### PR TITLE
DPM-105 redirect command

### PR DESCRIPTION
**Pull request by AndreasSaltsberg**
Adds new command `Command.REDIRECT_TO` that accepts two arguments:
`view` - (required) view where to redirect the user
`id` - (optional) id of the view. If not provided, user will be redirect to list view instead

### CODE REVIEW

**Comment by pipedrive-bot-eventsink** [comment]
### Main branch dependencies
[snyk vulnerabilities](https://app.snyk.io/org/gravity-byw/project/5013564b-fbd2-498f-ad71-ca01748463df)

File: README.md [hunk 1]
```diff
@@ -311,6 +312,23 @@ Closes an active modal window; applicable only for **custom modal**.
...
 
+### Redirect to
+
+Redirects user to specified view.
+
+**Parameters**
+
+| Parameter  | Type          | Description                                                                                                          | Notes                                                                                                                                                                                                |
+|------------|---------------|----------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| view       | View          | View where the user will be redirected to                                                                            | required<br/><br/>Supported views:<br/>View.DEALS<br/>View.LEADS<br/>View.ORGANIZATIONS<br/>View.CONTACTS<br/>View.CAMPAIGNS<br/>View.PROJECTS<br/>View.SETTINGS - redirects to custom settings page |
```
Comment (RuTsEST) [inline README.md:323]: Question (non-blocking): Should we mention here that the app should have its own custom settings page implemented?

**Review by RuTsEST — DISMISSED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (AndreasSaltsberg) [reply README.md]: I think this should be logical, but I can add link to custom settings page documentation, so it's clear what is meant by this?

**Review by RuTsEST — APPROVED** [review]

**Comment by pipedrive-bot-dora** [comment]
## Neodora Analysis ⚠


<details><summary><strong> ⚠ Unknown tribe name</strong></summary>
<br/>

**Message:** The property `owner` in `repository.json` must be a valid tribe name present in **mission-tracking**
**Repository file:** [repository.json](https://github.com/pipedrive/app-extensions-sdk/tree/c42ed738003fb839fe46f521714b434692e71013/repository.json)

<table>
<thead>
	<tr>
			<th>Owner status</th>
	</tr>
</thead>
<tbody>
		<tr>
				<td>'GRAVITY' is an unknown tribe name</td>
		</tr>
</tbody>
</table>


🔍 [How to fix it](https://neodora.pipedrive.tools/rules/tribeName)
<br/>
</details>

<details><summary><strong> ⚠ Missing cache in Github Workflow</strong></summary>
<br/>

**Message:** Github workflow should be using caching when step uses `actions/setup-node`

<table>
<thead>
	<tr>
			<th>Workflow file</th>
			<th>Job name</th>
			<th>Step index</th>
			<th>Actual cache value</th>
			<th>Expected cache value</th>
	</tr>
</thead>
<tbody>
		<tr>
				<td><a href="https://github.com/pipedrive/app-extensions-sdk/tree/c42ed738003fb839fe46f521714b434692e71013/.github/workflows/on-commit.yml">on-commit.yml</a></td>
				<td><code>lint</code></td>
				<td>2</td>
				<td><code>Not found</code></td>
				<td><code>npm</code></td>
		</tr>
</tbody>
</table>


🔍 [How to fix it](https://neodora.pipedrive.tools/rules/workflowCache)
<br/>
</details>


Commit [`c42ed73`](https://github.com/pipedrive/app-extensions-sdk/commit/c42ed738003fb839fe46f521714b434692e71013/)  Job [`93566`](https://jenkins.pipedrive.tools/job/neodora/93566/)

**Pull request merged**